### ISSUE TITLE
<Button href="xyz"> missing label

### ISSUE THREAD

**Issue by nictownsend** [carbon-design-system/carbon#5112]
## Environment

Chrome Mac OS X
DAP chrome plugin
Jan2020 archive, IBM Accessibility January 2020 ruleset

## Detailed description

> What version of the Carbon Design System are you using?

Carbon Components React 7.9.2

> What WCAG 2.1 checkpoint does the issue violate?

1.3.1 & 4.1.2

## Steps to reproduce the issue

https://codesandbox.io/s/dreamy-goldberg-z2miy

## Additional information

![image](https://user-images.githubusercontent.com/47386186/72814624-40805a00-3c5d-11ea-9978-20737bf805dc.png)

**Comment by joshblack** [comment]
Hi there! :wave: I'm unable to reproduce this issue using the given codesandbox while running the IBM Accessibility WCAG 2.1 September 2019 ruleset. Could you provide some more details to reproduce?

**Comment by nictownsend** [comment]
Hi Josh, I used the Jan2020 archive in DAP

**Comment by tw15egan** [comment]
Since the anchor buttons have a `role="button"`, looks like we need to add an `aria-label` to them. I can push up a quick fix

**Comment by joshblack** [comment]
@tw15egan when looking at this with @dakahn it seems like a bug on DAP, am going through the process of filing an issue there 👍

**Comment by joshblack** [comment]
Issued filed upstream at: https://ibm.biz/BdqDGA (can view behind IBM firewall)
